ethers-providers/src/transports/quorum.rs

```diff
@@ -742,41 +769,59 @@ mod tests {
             .add_providers(providers.clone())
             .quorum(Quorum::ProviderCount(5))
             .build();
-        assert_eq!(quorum.get_quorum_block_number().await.unwrap().as_u64(), 68);
```

Comment:
This formatting is incorrect, you need to use `cargo +nightly fmt` with this because they use nightly features for their configs.